brainiak/fcma/voxelselector.py

```diff
@@ -28,6 +28,7 @@
 from . import fcma_extension
 from . import cython_blas as blas
 import logging
+import pathos.multiprocessing
```

Comment:
Sorry I didn't make myself clear. It is not just about lambda functions. A lambda function without involving any other user-defined methods should probably be fine, but in my case, I have `_cross_validation_for_one_voxel` used in my lambda function. If I want to use the standalone multiprocessing, I will have to put `_cross_validation_for_one_voxel` outside the class and define it at the root level.